src/Datadog.Trace.ClrProfiler.Managed/Integrations/Testing/NUnitIntegration.cs

```diff
@@ -34,14 +34,13 @@ public static class NUnitIntegration
         private const string NUnitTestExecutionContextType = "NUnit.Framework.Internal.TestExecutionContext";
 
         private static readonly Vendors.Serilog.ILogger Log = DatadogLogging.GetLogger(typeof(NUnitIntegration));
-        private static readonly FrameworkDescription _runtimeDescription;
+        private static readonly FrameworkDescription RuntimeDescription;
```

Comment:
`NUnitIntegration` doesn't need to cache this anymore.

Reply:
Done